### DISCUSSION TITLE
Abandoned extensions.

### DISCUSSION THREAD

**Discussion by terabytesoftw** [yiisoft/yii2#19898] [Yii 22]
As i agree with @schmunk42, the reality is that there are many abandoned extensions that won't be updated for **Yii 2.2** Considering this, i believe we can tackle this challenge by identifying **essential extensions** and **updating** them to ensure they continue to function properly. Let's create a list of which ones would be crucial to **update**. i'm not saying this will happen overnight, but it seems like a promising path forward.

**Comment by tomaszkane** [comment]
Extensions for me:

Not abandoned:

- yiisoft/yii2-redis
- yiisoft/yii2-debug
- yiisoft/yii2-faker
- yiisoft/yii2-queue
- yiisoft/yii2-imagine
- yiisoft/yii2-gii

Not YiiSoft:

- bizley/migration
- codeception/module-yii2
- kartik-v/yii2-widget-datetimepicker

Adandoned:
- yii2tech/csv-grid
- yii2tech/spreadsheet
- yii2tech/html2pdf

Is it essential? Your choice :-)

**Comment by biladina** [comment]
Abandoned:

- yii2-swiftmailer

**Reply by samdark** [reply]
That's because swiftmailer is deprecated itself.

**Reply by mtangoo** [reply]
And new version of the successor was release days ago!

**Comment by bizley** [comment]
I'll update my packages for sure. But everyone needs to be prepared for the grim reality that many 3rd party extensions will not be updated, too much time left.

**Comment by rhertogh** [comment]
Additional (not yet mentioned above) extensions for me:
yiisoft/yii2-authclient
yiisoft/yii2-bootstrap
yiisoft/yii2-elasticsearch
yiisoft/yii2-httpclient
yiisoft/yii2-mongodb

**Comment by anovsiradj** [comment]
most of @kartik-v packages are used by almost every Yii user, so it cant be abandoned. <https://packagist.org/?query=yii2&page=2>
almost all of @yii2tech packages are archived, which is sad (i really like the behavior-trait, maybe sometime ill fork it in the future).

btw we can utilize <https://github.com/awesome-yii/list> to tell people recomendation about what package to use and what not.

**Comment by srakl** [comment]
Adandoned:
[yii2-recaptcha3](https://github.com/manchenkoff/yii2-recaptcha3)
[yii2tech/csv-grid](https://github.com/yii2tech/csv-grid)

**Comment by razvanphp** [comment]
Last updated 2 years ago:
[kartik-v/yii2-mpdf](https://github.com/kartik-v/yii2-mpdf)

**Reply by biladina** [reply]
but it still compatible with new PHP (I'm using 8.1) and Yii 2 version..
